graphql-cli/src/test/resources/expectedGenCode/client.bal

```diff
@@ -6,20 +6,36 @@ public isolated client class GraphqlClient {
     public isolated function init(string serviceUrl, ConnectionConfig config) returns graphql:ClientError? {
         http:ClientConfiguration httpClientConfig = {
             httpVersion: config.httpVersion,
-            http1Settings: {...config.http1Settings},
-            http2Settings: config.http2Settings,
             timeout: config.timeout,
             forwarded: config.forwarded,
             poolConfig: config.poolConfig,
-            cache: config.cache,
             compression: config.compression,
             circuitBreaker: config.circuitBreaker,
-            retryConfig: config.retryConfig,
-            responseLimits: config.responseLimits,
-            secureSocket: config.secureSocket,
-            proxy: config.proxy,
-            validation: config.validation
```

Comment:
Have we missed the `validation` field in the new record

Reply:
yes. Updated in https://github.com/ballerina-platform/graphql-tools/commit/b6d165f83dacb93282cd08d46309dce7a592f37d